string filter
---
int32 status
string msg
